src/hyperion/device_setup_plans/setup_panda.py

```diff
@@ -174,6 +172,10 @@ def setup_panda_for_flyscan(
     # Values need to be set before blocks are enabled, so wait here
     yield from bps.wait(group="panda-config", timeout=GENERAL_TIMEOUT)
 
+    LOGGER.info(f"PandA sequencer table has been set to: {str(table)}")
+    table_readback = yield from bps.rd(panda.seq[1].table)
+    LOGGER.info(f"PandA sequencer table readback is: {str(table_readback)}")
```

Comment:
Should: This is going to match the setpoint ideally so might be a bit noisy in the logs. If you think the bug is gone where we need to report it then remove it, otherwise can we put it as a debug?

Reply:
I'll put as debug now. When we're 100% sure there's no more issues with it I will remove